src/config/scrivito.js

```diff
@@ -1,16 +1,16 @@
 import * as Scrivito from "scrivito";
 
-const config = { tenant: process.env.SCRIVITO_TENANT, adoptUi: true };
+export function configureScrivito(options) {
+  const config = { tenant: process.env.SCRIVITO_TENANT, adoptUi: true };
 
-if (process.env.SCRIVITO_ORIGIN) {
-  config.origin = process.env.SCRIVITO_ORIGIN;
-}
+  if (process.env.SCRIVITO_ORIGIN) {
+    config.origin = process.env.SCRIVITO_ORIGIN;
+  }
 
-if (process.env.SCRIVITO_ENDPOINT) {
-  config.endpoint = process.env.SCRIVITO_ENDPOINT;
-}
+  if (process.env.SCRIVITO_ENDPOINT) {
+    config.endpoint = process.env.SCRIVITO_ENDPOINT;
+  }
 
-export function configureScrivito(options) {
   const priority = options?.priority;
   Scrivito.configure(priority ? { ...config, priority } : config);
```

Comment:
Since we're now building a new "config" on each call, we can modify it directly.

Something like:

```js
if (options?.priority) {
  config.priority = options?.priority;
}
```